ocs_ci/deployment/baremetal.py

```diff
@@ -320,28 +322,23 @@ def deploy(self, log_cli_level="DEBUG"):
                     self.mgmt_details[machine].get("cluster_name")
                     == constants.BM_DEFAULT_CLUSTER_NAME
                 ):
-                    secrets = [
-                        self.mgmt_details[machine]["mgmt_username"],
-                        self.mgmt_details[machine]["mgmt_password"],
-                    ]
-                    # Changes boot prioriy to pxe
-                    cmd = (
-                        f"ipmitool -I lanplus -U {self.mgmt_details[machine]['mgmt_username']} "
-                        f"-P {self.mgmt_details[machine]['mgmt_password']} "
-                        f"-H {self.mgmt_details[machine]['mgmt_console']} chassis bootdev pxe"
-                    )
-                    run_cmd(cmd=cmd, secrets=secrets)
-                    sleep(2)
-                    # Power On Machine
-                    cmd = (
-                        f"ipmitool -I lanplus -U {self.mgmt_details[machine]['mgmt_username']} "
-                        f"-P {self.mgmt_details[machine]['mgmt_password']} "
-                        f"-H {self.mgmt_details[machine]['mgmt_console']} chassis power cycle || "
-                        f"ipmitool -I lanplus -U {self.mgmt_details[machine]['mgmt_username']} "
-                        f"-P {self.mgmt_details[machine]['mgmt_password']} "
-                        f"-H {self.mgmt_details[machine]['mgmt_console']} chassis power on"
-                    )
-                    run_cmd(cmd=cmd, secrets=secrets)
+                    if self.mgmt_details[machine]["role"] == "bootstrap":
```

Comment:
I think we can add a new constant: `BOOTSTRAP_MACHINE = "bootstrap"` in ocs_ci/ocs/constants.py.

Reply:
Done